reactor-core/src/test/java/reactor/core/scheduler/SchedulersTest.java

```diff
@@ -734,34 +734,6 @@ public void immediateTaskIsExecuted() throws Exception {
 		assertThat((end - start) >= 1000).as("Timeout too long").isTrue();
 	}
 
-	@Test
-	public void immediateTaskIsSkippedIfDisposeRightAfter() throws Exception {
```

Comment:
This is no longer the case. It still could race, but now there's a few more instructions in `dispose()` so the task gets aborted. IMO it's not a _feature_ worth testing - the test was introduced in https://github.com/reactor/reactor-core/commit/6f3383d5a4fc533396a3522095cd4970024849e2 but one should not rely on a race to dispose tasks.